cupy/math/rounding.py

```diff
@@ -52,4 +52,4 @@
 
     .. seealso:: :func:`numpy.fix`
 
-    ''')
+    ''', support_complex=False)
```

Comment:
Please revert this because this function is `core.create_ufunc`.

Reply:
i see